docs/regression_test_api.rst

```diff
@@ -48,7 +48,7 @@ This provides the ReFrame internals with further control over the user's input,
 In essence, these builtins exert control over the test creation, and they allow adding and/or modifying certain attributes of the regression test.
 
 
-.. py:function:: reframe.core.pipeline.RegressionTest.parameter(values=None, inherit_params=False, filter_params=None)
+.. py:function:: RegressionTest.parameter(values=None, inherit_params=False, filter_params=None)
```

Comment:
I removed the prefix here because the link was not correct. It is prepending `reframe.core.pipeline` already.

Reply:
I think the cross-reference linking in the documentation is more important than a link to the source...